.coveragerc

```diff
@@ -41,6 +41,7 @@ omit =
 
     homeassistant/components/binary_sensor/arest.py
     homeassistant/components/binary_sensor/rest.py
+    homeassistant/components/binary_sensor/command_sensor.py
```

Comment:
This is a pure Python component and should not be excluded from coverage. Please remove this line.